minishift/kedge/auth-local.yml

```diff
@@ -1,16 +1,99 @@
+    AUTH_TENANT_SERVICEURL: aHR0cDovL2Y4dGVuYW50Ojgw
+  type: Opaque
+- name: auth-service-account-secrets
+  data:
+    oso-clusters.conf: ewogICAgImNsdXN0ZXJzIjogWwogICAgICAgIHsKICAgICAgICAgICAgIm5hbWUiOiJ1cy1lYXN0LTIiLAogICAgICAgICAgICAidXJsIjoiaHR0cHM6Ly9hcGkuc3RhcnRlci11cy1lYXN0LTIub3BlbnNoaWZ0LmNvbSIsCiAgICAgICAgICAgICJzZXJ2aWNlLWFjY291bnQtdG9rZW4iOiJmWDBuSDNkNjhMUTZTSzV3QkU2UWVLSjZYOEFaR1ZRTzNkR1FaWkVUYWtobWdtV0FxcjJLREZYRTY1S1V3Qk82OWFXb3EiLAogICAgICAgICAgICAidG9rZW4tcHJvdmlkZXItaWQiOiJmODY3YWMxMC01ZTA1LTQzNTktYTBjNi1iODU1ZWNlNTkwOTAiLAogICAgICAgICAgICAiYXV0aC1jbGllbnQtaWQiOiJhdXRoZWFzdDIiLAogICAgICAgICAgICAiYXV0aC1jbGllbnQtc2VjcmV0IjoiYXV0aGVhc3Qyc2VjcmV0IiwKICAgICAgICAgICAgImF1dGgtY2xpZW50LWRlZmF1bHQtc2NvcGUiOiJ1c2VyOmZ1bGwiCiAgICAgICAgfSwKICAgICAgICB7CiAgICAgICAgICAgICJuYW1lIjoidXMtZWFzdC0yYSIsCiAgICAgICAgICAgICJ1cmwiOiJodHRwczovL2FwaS5zdGFydGVyLXVzLWVhc3QtMmEub3BlbnNoaWZ0LmNvbSIsCiAgICAgICAgICAgICJzZXJ2aWNlLWFjY291bnQtdG9rZW4iOiJhazYxVDZSU0FhY1dGcnVoMXZaUDhjeVVPQnRRM0NodjFyZE9CZGRTdWM5bloyd0VjczgxREhYUk81NU5wSXBWUTh1aUgiLAogICAgICAgICAgICAidG9rZW4tcHJvdmlkZXItaWQiOiI4ODZjN2VhMy1lZjk3LTQ0M2QtYjM0NS1kZTk0Yjk0YmI2NWQiLAogICAgICAgICAgICAiYXV0aC1jbGllbnQtaWQiOiJhdXRoZWFzdDJhIiwKICAgICAgICAgICAgImF1dGgtY2xpZW50LXNlY3JldCI6ImF1dGhlYXN0MmFzZWNyZXQiLAogICAgICAgICAgICAiYXV0aC1jbGllbnQtZGVmYXVsdC1zY29wZSI6InVzZXI6ZnVsbCIKICAgICAgICB9CiAgICBdCn0=
+    service-account-secrets.conf: ewogICAgImFjY291bnRzIjogWwogICAgICAgIHsKICAgICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXdpdCIsCiAgICAgICAgICAgICJpZCI6IjVkZWM1ZmRiLTA5ZTMtNDQ1My1iNzNmLTVjODI4ODMyYjI4ZSIsCiAgICAgICAgICAgICJzZWNyZXRzIjpbIiQyYSQwNCRuSTd6N1JlNHBieC5WNXZ3bTE0bjUudmVsaEIubmJNZ3hkWjB2U29tV1Z4Y2N0MzR6Ykg5ZSJdCiAgICAgICAgfSwKICAgICAgICB7CiAgICAgICAgICAgICJuYW1lIjoiZmFicmljOC10ZW5hbnQiLAogICAgICAgICAgICAiaWQiOiJjMjExZjFiZC0xN2E3LTRmOGMtOWY4MC0wOTE3ZDE2Nzg4OWQiLAogICAgICAgICAgICAic2VjcmV0cyI6WyIkMmEkMDQkeW5xTS9zeUtNWW93TUluNWN5cUh1ZXZXbmZ6SVFxdHlZNG0uNjFCMDJxbHRZNVNPeUdJT2UiLCAiJDJhJDA0JHNiQy9BZlcyYzMzaHY4b3JHQS4xRC5MWGEvLklZNzZWV2hzZnF4Q1ZocmhGa0RmTDAvWEdLIl0KICAgICAgICB9LAogICAgICAgIHsKICAgICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LWplbmtpbnMtaWRsZXIiLAogICAgICAgICAgICAiaWQiOiIzNDFjMjgzZi0wY2Q3LTQ4YTgtOTI4MS00NTgzYWNlYjM2MTciLAogICAgICAgICAgICAic2VjcmV0cyI6WyIkMmEkMDQkaGJHSEFWS29ocGVEZ0h6YWZuTHdkTzRaemhFbjl1a1ZQLzZDYU90ZjVvM0J0cC5yNnRYVEciXQogICAgICAgIH0sCiAgICAgICAgewogICAgICAgICAgICAibmFtZSI6ImZhYnJpYzgtb3NvLXByb3h5IiwKICAgICAgICAgICAgImlkIjoiOTZlZTNmOTEtZjk4MC00MjlmLWE0ZTgtMDYwYzMyNTgwMzFkIiwKICAgICAgICAgICAgInNlY3JldHMiOlsiJDJhJDA0JGhiR0hBVktvaHBlRGdIemFmbkx3ZE80WnpoRW45dWtWUC82Q2FPdGY1bzNCdHAucjZ0WFRHIl0KICAgICAgICB9LAogICAgICAgIHsKICAgICAgICAgICAgIm5hbWUiOiJvbmxpbmUtcmVnaXN0cmF0aW9uIiwKICAgICAgICAgICAgImlkIjoiZjg2N2VjNzItMzE3MS00YjhmLThlZWMtOTBhMzJlYWI2ZTBiIiwKICAgICAgICAgICAgInNlY3JldHMiOlsiaklSNUZwWXEwUVVVek1FS3FpQUlWbG9OQkNMM3Yxbk9QeHE5V20wN3ZUc0poS0dOQlJXV3Nka25LNHg0ZWwzIl0KICAgICAgICB9CiAgICBdCn0=
```

Comment:
Why not re-using the default config here? If these files are not mounted then https://github.com/fabric8-services/fabric8-auth/tree/master/configuration/conf-files will be used. The default config is preferable in dev env.
The health check will fail if the files are not mounted and service is not run in Dev Mode (AUTH_DEVELOPER_MODE_ENABLED = true) though.

Reply:
I had issues running without these files being mounted, but let me give it a try and get back to this comment.